.gitignore

```diff
@@ -5,6 +5,7 @@
 /.pnp
 .pnp.js
 .prettierrc
+.history
```

Comment:
/history
chyba powinno zadziałać

Reply:
posprzątam w wolnej chwili ;) THX! :)

Reply:
Wygląda na to, że pliki są dalej dodawane, tylko pod innymi nazwami. Proponuję dodać to .gitignore ".history/*" i usunąc pliki jeszcze raz